code/modules/sanity/breakdowns.dm

```diff
@@ -0,0 +1,525 @@
+			"You know that only salvation from your sins is [objectname]."
+		))
+		to_chat(holder.owner, message)
+
+/datum/breakdown/common/obsession/occur()
+	for(var/stat in ALL_STATS)
+		holder.owner.stats.addTempStat(stat, -5, INFINITY, "Obsession")
+	return ..()
+
+/datum/breakdown/common/obsession/conclude()
+	for(var/stat in ALL_STATS)
+		holder.owner.stats.removeTempStat(stat, "Obsession")
+	..()
+
+
+
+#define KLEPTOMANIA_COOLDOWN rand(30 SECONDS, 60 SECONDS)
+
+/datum/breakdown/common/kleptomania
+	name = "Kleptomania"
+	duration = 20 MINUTES
+	restore_sanity_post = 50
+	var/pickup_time = 0
+
+	start_messages = list(
+		"You feel need to hold something, that you shouldn't...",
+		"You feel like others don't value what they have, but you on the other side...",
+		"You feel like everything should be in your possession...",
+		"You feel like everything can be yours, just a small effort needed...",
+		"You feel like some things have strong aura around them, it won't hurt to take them for a while..."
+	)
+	end_messages = list(
+		"You feel easier without stealing things now."
+	)
+
+/datum/breakdown/common/kleptomania/update()
+	. = ..()
+	if(!.)
+		return
+	if(world.time >= pickup_time)
+		pickup_time = world.time + KLEPTOMANIA_COOLDOWN
+		var/list/obj/item/candidates = oview(1, holder.owner)
+		while(candidates.len)
+			var/obj/item/I = pick(candidates)
+			if(!istype(I) || !I.Adjacent(holder.owner) || !I.pre_pickup(holder.owner))
+				candidates -= I
+				continue
+			if(!holder.owner.put_in_hands(I) && prob(50))
+				holder.owner.unEquip(holder.owner.get_inactive_hand())
+				holder.owner.put_in_hands(I)
+			break
+
+
+
+/datum/breakdown/common/signs
+	//name = "Signs"
+	restore_sanity_post = 70
+	var/message
+
+	start_messages = list(
+		"You feel like fabric of reality is visible to you...",
+		"You feel that truth is hidden somewhere, right in your mind...",
+		"You feel like your mind have spoken to you, after centuries of silence...",
+		"You feel like you were blind, but now you see...",
+		"You feel like universe itself speaking to you..."
+	)
+	end_messages = list(
+		"The truth have spoken. You feel it again. Melody of sounds is back to you."
+	)
+
+/datum/breakdown/common/signs/New()
+	..()
+	message = "Etiam tempor orci eu lobortis elementum nibh tellus molestie"
+
+/datum/breakdown/common/signs/update()
+	. = ..()
+	if(!.)
+		return
+	if(!prob(40))
+		return
+	var/list/words = splittext(message, " ")
+	var/phrase_len = rand(1,2)
+	var/phrase_pos = rand(1, words.len - phrase_len)
+	to_chat(holder.owner,"...[jointext(words, " ", phrase_pos, phrase_pos + phrase_len + 1)]...")
+
+/datum/breakdown/common/signs/occur()
+	RegisterSignal(holder.owner, COMSIG_HUMAN_SAY, .proc/check_message)
+	return ..()
+
+/datum/breakdown/common/signs/conclude()
+	UnregisterSignal(holder.owner, COMSIG_HUMAN_SAY)
+	..()
+
+/datum/breakdown/common/signs/proc/check_message(_, msg)
```

Comment:
Is `_` valid arg name?

Reply:
Yes, it works. I use `_` for unused args that are required by signals.